k8s/testdata/expected-vulndb.json

```diff
@@ -1 +1 @@
:"1.11.0"},{"fixed":"1.11.2"}]}]}],"references":[{"type":"ADVISORY","url":"https://github.com/kubernetes/kubernetes/issues/65750"},{"type":"ADVISORY","url":"https://www.cve.org/cverecord?id=CVE-2018-1002101"}]},{"id":"CVE-2018-1002100","modified":"2018-03-16T19:24:46Z","published":"2018-03-16T19:24:46Z","summary":"Kubectl copy doesn't check for paths outside of it's destination directory.","details":"In Kubernetes versions 1.5.x, 1.6.x, 1.7.x, 1.8.x, and prior to version 1.9.6, the kubectl cp command insecurely handles tar data returned from the container, and can be caused to overwrite arbitrary local files.","affected":[{"package":{"ecosystem":"kubernetes","name":"k8s.io/kubectl"},"severity":[{"type":"CVSS_V3","score":"CVSS:3.0/AV:N/AC:H/PR:H/UI:R/S:U/C:N/I:H/A:N"}],"ranges":[{"type":"SEMVER","events":[{"introduced":"1.5.0"},{"fixed":"1.9.6"}]}]}],"references":[{"type":"ADVISORY","url":"https://github.com/kubernetes/kubernetes/issues/61297"},{"type":"ADVISORY","url":"https://www.cve.org/cverecord?id=CVE-2018-1002100"}]},{"id":"CVE-2017-1002102","modified":"2018-03-05T20:55:20Z","published":"2018-03-05T20:55:20Z","summary":"atomic writer volume handling allows arbitrary file deletion in host filesystem","details":"In Kubernetes versions 1.3.x, 1.4.x, 1.5.x, 1.6.x and prior to versions 1.7.14, 1.8.9 and 1.9.4 containers using a secret, configMap, projected or downwardAPI volume can trigger deletion of arbitrary files/directories from the nodes where they are running.","affected":[{"package":{"ecosystem":"kubernetes","name":"k8s.io/kubelet"},"severity":[{"type":"CVSS_V3","score":"CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:L/A:H"}],"ranges":[{"type":"SEMVER","events":[{"introduced":"1.3.0"},{"fixed":"1.7.14"},{"introduced":"1.8.0"},{"fixed":"1.8.9"},{"introduced":"1.9.0"},{"fixed":"1.9.4"}]}]}],"references":[{"type":"ADVISORY","url":"https://github.com/kubernetes/kubernetes/issues/60814"},{"type":"ADVISORY","url":"https://www.cve.org/cverecord?id=CVE-2017-1002102"}]},{"id":"CVE-2017-1002101","modified":"2018-03-05T20:53:58Z","published":"2018-03-05T20:53:58Z","summary":"subpath volume mount handling allows arbitrary file access in host filesystem","details":"In Kubernetes versions 1.3.x, 1.4.x, 1.5.x, 1.6.x and prior to versions 1.7.14, 1.8.9 and 1.9.4 containers using subpath volume mounts with any volume type (including non-privileged pods, subject to file permissions) can access files/directories outside of the volume, including the host's filesystem.","affected":[{"package":{"ecosystem":"kubernetes","name":"k8s.io/kubernetes"},"severity":[{"type":"CVSS_V3","score":"CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"}],"ranges":[{"type":"SEMVER","events":[{"introduced":"1.3.0"},{"fixed":"1.7.14"},{"introduced":"1.8.0"},{"fixed":"1.8.9"},{"introduced":"1.9.0"},{"fixed":"1.9.4"}]}]}],"references":[{"type":"ADVISORY","url":"https://github.com/kubernetes/kubernetes/issues/60813"},{"type":"ADVISORY","url":"https://www.cve.org/cverecord?id=CVE-2017-1002101"}]}]
```

Comment:
Are these expected changes?

Reply:
No, the whole CI test fails, so I just fixed it and it has nothing to do with the changes to the debian tracker.

Reply:
https://github.com/aquasecurity/vuln-list-update/actions/runs/6817001043/job/18539561269

Reply:
Give me some time to understand why k8s test broke.

Reply:
Thanks for information.
I think we don't need to merge this change with your changes.
It's also strange that test stops working correctly without changes in `k8s` package.

I create #255 for this case.

Reply:
I dropped the commit that made the changes to this k8s.

Reply:
https://github.com/aquasecurity/vuln-list-update/pull/255 has been merged

Reply:
I rebased the main branch.